From 3e35f5d4f85f32a05d7d4353f4295468028e4b85 Mon Sep 17 00:00:00 2001
From: "Nguyen Van Tu [FGA.AIS]" <TuNV16@fsoft.com.vn>
Date: Fri, 25 Jan 2019 14:33:59 +0700
Subject: [PATCH 624/628] ARM: DTS: r8a7743: revert watchdog node to keep
 community name

	- Revert 2 commits 38f75d00cd0a1120cf04e44ffb646d56f04e46d1
	  and 886f6b3172ffc8ceda348372c02844447d9fedb8 to follow
	  changes of community

Signed-off-by: Nguyen Van Tu [FGA.AIS] <TuNV16@fsoft.com.vn>
---
 arch/arm/boot/dts/iwg20d-q7-common.dtsi |  4 ----
 arch/arm/boot/dts/r8a7743-iwg20m.dtsi   |  5 +++++
 arch/arm/boot/dts/r8a7743.dtsi          | 18 ++++++++++--------
 3 files changed, 15 insertions(+), 12 deletions(-)

diff --git a/arch/arm/boot/dts/iwg20d-q7-common.dtsi b/arch/arm/boot/dts/iwg20d-q7-common.dtsi
index 471eceb..b749864 100644
--- a/arch/arm/boot/dts/iwg20d-q7-common.dtsi
+++ b/arch/arm/boot/dts/iwg20d-q7-common.dtsi
@@ -213,10 +213,6 @@
 	};
 };
 
-&wdt0{
-	status = "okay";
-};
-
 &xhci {
 	status = "okay";
 };
diff --git a/arch/arm/boot/dts/r8a7743-iwg20m.dtsi b/arch/arm/boot/dts/r8a7743-iwg20m.dtsi
index 1d3e950..d364685 100644
--- a/arch/arm/boot/dts/r8a7743-iwg20m.dtsi
+++ b/arch/arm/boot/dts/r8a7743-iwg20m.dtsi
@@ -91,6 +91,11 @@
 	};
 };
 
+&rwdt {
+	timeout-sec = <60>;
+	status = "okay";
+};
+
 &sdhi0 {
 	pinctrl-0 = <&sdhi0_pins>;
 	pinctrl-names = "default";
diff --git a/arch/arm/boot/dts/r8a7743.dtsi b/arch/arm/boot/dts/r8a7743.dtsi
index 3a726db..0523bf3 100644
--- a/arch/arm/boot/dts/r8a7743.dtsi
+++ b/arch/arm/boot/dts/r8a7743.dtsi
@@ -116,6 +116,15 @@
 			reg = <0 0xe6160000 0 0x100>;
 		};
 
+		rwdt: watchdog@e6020000 {
+			compatible = "renesas,r8a7743-wdt",
+				     "renesas,rcar-gen2-wdt";
+			reg = <0 0xe6020000 0 0x0c>;
+			clocks = <&mstp4_clks R8A7743_CLK_RWDT>;
+			power-domains = <&cpg_clocks>;
+			status = "disabled";
+		};
+
 		gic: interrupt-controller@f1001000 {
 			compatible = "arm,gic-400";
 			#interrupt-cells = <3>;
@@ -132,13 +141,6 @@
 			power-domains = <&cpg_clocks>;
 		};
 
-		wdt0: watchdog@e6020000 {
-			compatible = "renesas,r8a7743-wdt", "renesas,rcar-gen2-wdt";
-			reg = <0 0xe6020000 0 0x0c>;
-			clocks = <&mstp4_clks R8A7743_CLK_RWDT>;
-			timeout-sec = <60>;
-		};
-
 		gpio0: gpio@e6050000 {
 			compatible = "renesas,gpio-r8a7743",
 				     "renesas,rcar-gen2-gpio";
@@ -1446,7 +1448,7 @@
 					R8A7743_CLK_RWDT R8A7743_CLK_IRQC
 					R8A7743_CLK_INTC_SYS
 				>;
-				clock-output-names = "wdt0", "irqc", "intc-sys";
+				clock-output-names = "rwdt", "irqc", "intc-sys";
 			};
 			mstp5_clks: mstp5_clks@e6150144 {
 				compatible = "renesas,r8a7743-mstp-clocks",
-- 
2.7.4

